dino_catalog/dinoparser.rb

```diff
@@ -0,0 +1,114 @@
+class DinoParser
+  attr_reader :files, :dinosaurs
+
+  def initialize(files = [])
+    @files = files
+    @dinosaurs = []
+  end
+
+  def add(dino)
+    @dinosaurs << Dinosaur.new(standardize(dino))
+  end
+
+  def find_bipeds
+    @dinosaurs.select(&:is_biped?)
```

Comment:
Don't prepend boolean methods with `is_`. It's a Javaism. The Ruby corollary is just the `?` suffix.